docs/readthedocs/_static/bibliography/bibtexAll.bib

```diff
@@ -15,7 +15,7 @@ @Article{Maleki.2017
 @MISC{Itaca_Sun,
 author = {Itacanet},
 title = {The Sun As A Source Of Energy},
-howpublished={\url{https://www.itacanet.org/the-sun-as-a-source-of-energy/part-3-calculating-solar-angles/}}
```

Comment:
Not quite happy with that. We should find a new source, preferably not a web source that can just disappear at any point in time.